docs/references/strategies/languages/python/setuptools.md

```diff
@@ -46,9 +46,14 @@ from both files.
 * Python requirements files and setup.py files do not provide any data about edges between dependencies.
 * Requirements files can be completely different than an existing setup.py specification, as there is no built-in
 synchronization between them.
-* Since we don't completely parse the python files, any programmability done in the `setup.py` file will hide the
-true `install_requires` list from our view.  However, we do catch *variables* named `install_requires` as well, as
-long as they are declared earlier in the file than the `install_requires` keyword argument to `setup`.
+* Since we don't actually run `setup.py` or do our own interpretation of the Python code therein, `install_requires` not defined as a literal array of string literals done in the `setup.py` file will hide the
+true `install_requires` list from our view. For example, we don't have a way to find `install_requires` set up this way:
+```python
+a = ['package1==1.0.0']
+b = ['package2==2.0.0']
+install_requires = a + b
+```
+However, we do catch *variables* named `install_requires`, as long as they are declared earlier in the file than the `install_requires` keyword argument to `setup`.
```

Comment:
Nit: if you indent these lines by three spaces then they get included in the list item and render a bit more nicely

```suggestion
* Since we don't actually run `setup.py` or do our own interpretation of the Python code therein, `install_requires` not defined as a literal array of string literals done in the `setup.py` file will hide the
   true `install_requires` list from our view. For example, we don't have a way to find `install_requires` set up this way:
   ```python
   a = ['package1==1.0.0']
   b = ['package2==2.0.0']
   install_requires = a + b
   ```
   However, we do catch *variables* named `install_requires`, as long as they are declared earlier in the file than the `install_requires` keyword argument to `setup`.
```

Reply:
Sorry, my suggested edits didn't quite work because there was a code-block in the suggested edit, but hopefully the idea still makes sense

Reply:
Thanks for catching that! I think I was able to get it to look better and also did some rewording to make sure `install_requires` is introduced before I start giving more details about it.